src/Init/Coe.lean

```diff
@@ -27,11 +27,11 @@ class CoeTail (α : Sort u) (β : Sort v) where
 class CoeHTCT (α : Sort u) (β : Sort v) where
   coe : α → β
 
-class CoeDep (α : Sort u) (_ : α) (β : Sort v) where
+class CoeDep (α : Sort u) (a : α) (β : Sort v) where
```

Comment:
@larsk21 I wonder if we should exclude class signatures by default

Reply:
After seeing this use case, yes, probably. But the same issue occurs with `structure` and `inductive` signatures as well, doesn't it?

Reply:
see #1191

Reply:
> But the same issue occurs with `structure` and `inductive` signatures as well, doesn't it?

That's a good question. It could be argued that it is likelier for classes where parameters that are exclusively for directing typeclass resolution sounds quite reasonable, whereas "faux-typed" structures such as the single `KeyedDeclsAttribute.Def` from this PR seem more edge case (and often may as well be `def`s, in which case we can't exclude them anyway).